tests/agenda_test.py

```diff
@@ -42,6 +43,33 @@ def test_create_agenda():
     response = requests.request("POST", url + 'api/v1/doctors/agenda', headers=headers,data=payload)
 
 
+    # Check the response
+    assert response.status_code == 200
+    assert response.json()['detail']['status'] == 'ok'
+    assert response.json()['detail']['status_code'] == 0
+
+
+# Test delete agenda - Success Case
+def test_delete_agenda():
+    
+    url = settings.api_test_url
+    
+    response = login_for_tests('doctor')
+    
+    # Check the response
+    assert response.status_code == 200
+    assert response.json()['access_token'] != None
+
+    headers = {
+        'Authorization': 'Bearer ' + response.json()['access_token'],
+        'Content-Type': 'application/json'
+    }
+
+    agenda_id = CrudHelper().get_last_result_id(Agenda)
+    
+    response = requests.request("DELETE", url + f'api/v1/doctors/agenda/{agenda_id}', headers=headers)
+
+
     # Check the response
     assert response.status_code == 200
     assert response.json()['detail']['status'] == 'ok'
```

Comment:
**Suggestion:** In the `test_delete_agenda` function, you should also check if the agenda was actually deleted from the database. You can do this by trying to fetch the deleted agenda after the delete operation and asserting that it doesn't exist.
```suggestion
def test_delete_agenda():

    url = settings.api_test_url

    response = login_for_tests('doctor')

    headers = {
        'Authorization': 'Bearer ' + response.json()['access_token'],
        'Content-Type': 'application/json'
    }

    agenda_id = CrudHelper().get_last_result_id(Agenda)

    response = requests.request("DELETE", url + f'api/v1/doctors/agenda/{agenda_id}', headers=headers)

    assert response.status_code == 200
    assert response.json()['detail']['status'] == 'ok'

    # Check if the agenda was actually deleted
    response = requests.request("GET", url + f'api/v1/doctors/agenda/{agenda_id}', headers=headers)
    assert response.status_code == 404
```